src/commands/createNewProject/dotnetSteps/DotnetRuntimeStep.ts

```diff
@@ -28,6 +28,12 @@ export class DotnetRuntimeStep extends AzureWizardPromptStep<IProjectWizardConte
         } else if (runtimes.length === 1) {
             // No need to prompt if it only supports one
             setWorkerRuntime(context, runtimes[0]);
+        } else if (context.targetNetwork) {
+            // if a targetNetwork was provided from createNewProject
+            const workerRuntime = runtimes.find(runtime => runtime.targetFramework === context.targetNetwork);
+            if (workerRuntime) {
+                setWorkerRuntime(context, workerRuntime);
+            }
```

Comment:
I feel like this should throw an error instead of falling through. Something like `Unrecognized target framework "{0}". Available frameworks: "{1}".`